### DISCUSSION TITLE
UseGpu

### DISCUSSION THREAD

**Discussion by PopescuGeorge94** [Mapsui/Mapsui#2176] [Q&A]
When I used MapControl.UseGPU = true in .Net MAUI, the screen becomes white, only when change the tab content. Any idea why?

**Comment by pauldendulk** [comment]
This is a bug in SkiaSharp. Hopefully this will be fixed in the next version, which will be released ... *checking nuget.org* ... 3 hours ago 😊

![image](https://github.com/Mapsui/Mapsui/assets/963462/b8864047-5164-418f-af9e-a7c24104903e)

**Reply by PopescuGeorge94** [reply]
Thanks :)

**Comment by inforithmics** [comment]
pull request
https://github.com/Mapsui/Mapsui/pull/2086
